src/Util/AstHelper.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Roave\BetterReflection\Util;
+
+use Closure;
+use PhpParser\Node;
+
+use Roave\BetterReflection\BetterReflection;
+use Roave\BetterReflection\Reflection\Reflection;
+use Roave\BetterReflection\Reflection\ReflectionClass;
+use Roave\BetterReflection\Reflection\ReflectionFunction;
+
+use PhpParser\Node\Expr\Closure as ClosureNode;
+use PhpParser\Node\Expr\ClosureUse as ClosureUseNode;
+use PhpParser\Node\Expr\ArrowFunction as ArrowFunctionNode;
+
+use PhpParser\NodeTraverser;
+use PhpParser\NodeVisitor\FindingVisitor;
+use PhpParser\PrettyPrinter\Standard as StandardPrettyPrinter;
+use PhpParser\PrettyPrinterAbstract;
+use RuntimeException;
+
+final class AstHelper {
```

Comment:
We should probably rename this to get rid of the `Helper` naming smell (after all, all code is a "helper" :) )

Perhaps this could be named `ExtractAst`? Usage would then be `ExtractAst#fromClass()`, `ExtractAst#fromFunction()`, etc...